.release-please-manifest.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Release please uses this file as a manifest to know what are the current released versions.
I used the current releases versions of the CLI and providers, but we can update it later on